doc/ref_model/chapters/appendix-a.md

```diff
@@ -70,6 +71,22 @@ Without arguing for or against SR-IOV, CNTT provides the following anecdotes whi
 End of policies.
 
 <a name="A.6"></a>
+## A.6 VNF Design and Implementation Requirements
+This section specifies a set of VNF Design and Implementation Requirements. 
```

Comment:
```suggestion
This section specifies a set of VNF Design and Implementation Guidelines.   While not mandatory, VNF vendors are strongly encouraged to comply with these guidelines.
```

For the most part, the CNTT does not have a mechanism for enforcing them as 'must' requirements.  Because they are mostly un-testable at RC time, it is likely better to specify them as suggested/strongly encouraged tasks.

Reply:
@kedmison I think we need both requirements and guidelines.